cmd/monaco/download/download_command.go

```diff
@@ -16,140 +16,115 @@ package download
-		RunE: func(cmd *cobra.Command, args []string) error {
-			return fmt.Errorf("'direct' or 'manifest' sub-command is required")
-		},
-	}
-
-	getDownloadConfigsCommand(fs, command, downloadCmd)
-
-	if featureflags.Entities().Enabled() {
-		getDownloadEntitiesCommand(fs, command, downloadCmd)
-	}
-
-	return downloadCmd
-}
-
-func getDownloadConfigsCommand(fs afero.Fs, command Command, downloadCmd *cobra.Command) {
 	var project, outputFolder string
 	var forceOverwrite bool
 	var specificApis []string
 	var specificSettings []string
+	var specificEnvironment string
 	var onlyAPIs bool
 	var onlySettings bool
+	var token string
+	var oAuthClientID, oAuthClientSecret string
+	var manifestFile string
+	var url string
 
-	manifestDownloadCmd := &cobra.Command{
-		Use:     "manifest [manifest file] [environment to download]",
-		Aliases: []string{"m"},
-		Short:   "Download configuration from Dynatrace via a manifest file",
-		Example: `monaco download manifest.yaml some_environment_from_manifest`,
-		Args: func(cmd *cobra.Command, args []string) error {
-			if len(args) != 2 || args[0] == "" || args[1] == "" {
-				return fmt.Errorf(`manifest and environment name have to be provided as positional arguments`)
-			}
-			return nil
-		},
-		ValidArgsFunction: completion.DownloadManifestCompletion,
-		RunE: func(cmd *cobra.Command, args []string) error {
-			m := args[0]
-			specificEnvironment := args[1]
-			options := manifestDownloadOptions{
-				manifestFile:            m,
-				specificEnvironmentName: specificEnvironment,
-				downloadCommandOptions: downloadCommandOptions{
-					downloadCommandOptionsShared: downloadCommandOptionsShared{
-						projectName:    project,
-						outputFolder:   outputFolder,
-						forceOverwrite: forceOverwrite,
-					},
-					specificAPIs:    specificApis,
-					specificSchemas: specificSettings,
-					onlyAPIs:        onlyAPIs,
-					onlySettings:    onlySettings,
-				},
-			}
+	downloadCmd = &cobra.Command{
+		Use:   "download --manifest <manifest.yaml> --token <API_TOKEN_ENV_VAR_NAME>",
+		Short: "Download configuration from Dynatrace",
+		Long: `Download configuration from Dynatrace
 
-			return command.DownloadConfigsBasedOnManifest(fs, options)
-		},
-	}
+Either downloading based on an existing manifest, or define an URL pointing to an environment to download configuration from.`,
 
-	directDownloadCmd := &cobra.Command{
-		Use:     "direct [URL] [TOKEN_NAME]",
-		Aliases: []string{"d"},
-		Short:   "Download configuration from a Dynatrace environment specified on the command line",
-		Example: `monaco download direct https://environment.live.dynatrace.com API_TOKEN_ENV_VAR_NAME`,
-		Args: func(cmd *cobra.Command, args []string) error {
-			if len(args) != 2 || args[0] == "" || args[1] == "" {
-				return fmt.Errorf(`url and token have to be provided as positional argument`)
+		Example: `- monaco download --manifest manifest.yaml --token API_TOKEN_ENV_VAR_NAME
```

Comment:
```suggestion
		Example: `- monaco download --manifest manifest.yaml --environment dev-env
```

Reply:
The whole help section is changed. please take a look